src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1446,7 +1464,7 @@ MulticopterPositionControl::task_main()
 							float i = _params.thr_min;
 
 							if (reset_int_z_manual) {
-								i = _manual.z;
+								i = _params.thr_hover;
```

Comment:
From a comment (far) above:  On the plus side, if the user sets the hover throttle parameter to the correct hover thrust level, transitions are smoother. (that is from flight testing observations)
This is the main reason for adding the hover thrust parameter; to make the transition from manual to ALTCTL occur at center stick == zero vertical speed == hover.